### ISSUE TITLE
Effects and emotions

### ISSUE THREAD

**Issue by Combie81** [3ative/Making-TTS-Better#1]
Can you add emotions to the random selections?  If so, how? I tried but it seems to read them out rather than convert and action them.

          [
          "Turning off Adults light... ",
          "Adults Light off... ",
          "Adults Light powering down... ",
          "Shutting down Adults lighting... ",
          "Adults light is now off... ",
          ]|random +[

          "<amazon:emotion name="excited" intensity="high">Is it snuggle time?</amazon:emotion>",
          "<amazon:effect name="whispered">ooh thats dark.</amazon:effect>",
          "<amazon:effect name="whispered">I hope you can see in the dark.</amazon:effect>",
          "<amazon:effect name="whispered">do you feel frightened too?</amazon:effect>",
          "<amazon:effect name="whispered">the world is a dark scary place.</amazon:effect>",
          "<amazon:effect name="whispered">please turn it on again, Im scared of the dark.</amazon:effect>",
          ]|random}}'

**Comment by 3ative** [comment]
Use 'announce' not TTS

**Comment by 3ative** [comment]
Also, check your punctuation.
E.G. This works: 
```yaml
"<amazon:effect name ='whispered'>this is a test</amazon:effect>"
```

**Comment by Combie81** [comment]
Thanks for the incredibly quick replies :)

No joy - I get the following errors:
"Incorrect type. Expected "object"." - line 1
"bad indentation of a mapping entry" - lines 18 & 20
"bad indentation of a sequence entry" - line 19
"end of the stream or a document seperator is expected" - lines 21-25
"unecpected end of the stream within a single quoted scalar" - line 26

tts_adults_light_off:
  alias: TTS Adults Light off
  sequence:
    - service: notify.alexa_media
      data_template:
        target:
          - "{{ states.sensor.last_alexa.state }}"
        data:
          type: announce
        message: '{{
          [
          "Turning off Adults light... ",
          "Adults Light off... ",
          "Adults Light powering down... ",
          "Shutting down Adults lighting... ",
          "Adults light is now off... ",
          ]|random +[

          "<amazon:emotion name='excited' intensity='high'>Is it snuggle time?</amazon:emotion>",
          "<amazon:effect name='whispered'>ooh thats dark.</amazon:effect>",
          "<amazon:effect name='whispered'>I hope you can see in the dark.</amazon:effect>",
          "<amazon:effect name='whispered'>do you feel frightened too?</amazon:effect>",
          "<amazon:effect name='whispered'>the world is a dark scary place.</amazon:effect>",
          "<amazon:effect name='whispered'>please turn it on again, Im scared of the dark.</amazon:effect>",
          ]|random}}'

**Comment by 3ative** [comment]
Oh hang a minute.... you're doing this with scripts and not Node-Red, right?

Hmmm, I could only get those "amazon emotions" working in Node-Red.

Lemme have a think about it and I'll get back to you

**Comment by Combie81** [comment]
Yeah, I've not taken the plunge down that rabbit hole yet!  I really should look into it but setup isn't really needing more complexity atm..... hopefully I'll get a bit of time to go through some of Drzzs tutorials on it :P

**Comment by Combie81** [comment]
Just had a thought and am wondering if an escape char is required? like \ or ^ or whatever it uses?

**Comment by 3ative** [comment]
Sorry for the slow replies, work has taken a step up lately and It's been hectic here.
- I haven't had a spare moment to work on this... I haven't forgotten about you thou :)

**Comment by Combie81** [comment]
I know the feeling! There's no rush - it's your free time so I'm glad of any help :)
